public/video-ui/src/reducers/localUploadsReducer.js

```diff
@@ -0,0 +1,19 @@
+const EMPTY = { handle: null, progress: 0, total: 0 };
```

Comment:
Is this reducer meant for single uploads and the uploads reducer for all the uploads? If so, then naming is misleading.